src-tauri/src/aux_functions/graph_init.rs

```diff
@@ -16,50 +16,94 @@ To mock a graph, one can call the function load_graph() with a vector of Neighbo
 and location hashmap generated by the mock_generator module.
 */
 
+pub fn add_node_to_graph_if_not_exists(graph: &mut Graph, node_id: u32) {
+    let name: String = node_id.to_string();
+    if !graph.contains_node(name.clone()) {
+        graph.add_node(name.clone());
+    }
+}
+
+pub fn as_key(node_1: u32, node_2: u32) -> (u32, u32) {
```

Comment:
What's the purpose of this?

Reply:
I'm using a hashmap to store edges in format [node, node] : [snr, timestamp] and didn't want to deal with collisions between node ids in the form of (A,B) and (B, A). That function standardizes the ordering of the node ids in the key.